rasgotransforms/rasgotransforms/testing_utils.py

```diff
@@ -0,0 +1,131 @@
+from pathlib import Path
```

Comment:
Organization recommendation: we may want move this file into the `/tests/` folder.

Since this is a public repo, there's value in keeping the test functions separate from the transform library

Reply:
The Black formatter yelled at me because everything in the `tests` dir should be a test ending in `_test.py`

Reply:
just call that file `test.py` then?